content/docs/get-started/aws/review-project.md

```diff
@@ -105,16 +105,29 @@ class Program
 This Pulumi program creates an S3 bucket and exports the name of the bucket.
 
 {{% lang python %}}
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python3 -m venv venv
 ```
 
+Activate the environment in Linux and macOS:
+
 ```bash
 $ source venv/bin/activate
 ```
 
+Activate the environment in Windows:
+
+```bat
+> venv\Scripts\activate
+```
+
+Install dependencies:
+
 ```bash
 $ pip3 install -r requirements.txt
```

Comment:
If we are going the `Python -m` route - should this be `python3 -m pip`?

Reply:
I believe it should be `pip install ...` since you have already instructed the user to activate the virtual environment, and `pip` is bootstrapped in by default.

https://docs.python.org/3/library/venv.html (see discussion of `—without-pip`)

Related: looks like the activate command for Windows is missing in the site preview.